test/parallel/test-https-client-renegotiation-limit.js

```diff
@@ -21,11 +21,15 @@
 
 'use strict';
 const common = require('../common');
-if (!common.hasCrypto)
+if (!common.hasCrypto) {
   common.skip('missing crypto');
+}
+
+const { opensslCli } = require('../common/crypto');
 
-if (!common.opensslCli)
+if (!opensslCli) {
   common.skip('node compiled without OpenSSL CLI.');
+}
```

Comment:
```suggestion
```

I think this can be removed, `opensslCli` does not seem to be used. Feel free to ignore or move to another commit/PR.

Reply:
I'll leave these as they are for now.